src/code/z_skelanime.c

```diff
@@ -1838,6 +1838,7 @@ void SkelAnime_UpdateTranslation(SkelAnime* skelAnime, Vec3f* diff, s16 angle) {
     f32 sin;
     f32 cos;
 
+    // If `ANIM_FLAG_UPDATE_XZ` behaved as expected, it flag would also be checked here
```

Comment:
proofread
maybe "it would" ?

Reply:
yeah I had one sentence and decided to change it to another but left a word in . thanks